src/ShallowTraversal.js

```diff
@@ -29,7 +29,7 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   const classes = propsOfNode(node).className || '';
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  return ` ${classes.trim()} `.indexOf(` ${className} `) > -1;
```

Comment:
Also, DOM API's like `document.querySelector` will ignore leading/trailing whitespace but not significant whitespace, which is kind of what I was following here. If you have significant whitespace in your class names (which you probably shouldn't anyways) you should test explicitly for that.

What do you think?